readme.md

```diff
@@ -154,7 +154,21 @@ Defer showing the notification to after the process has exited.
 Type: `string`\
 Default: [See above screenshot](https://github.com/yeoman/update-notifier#update-notifier-)
 
-Message that will be shown when an update is available.
+Message that will be shown when an update is available, support [`pupa`](https://github.com/sindresorhus/pupa) template.
+
+Available placeholders:
+
+- `{name}` - Package name.
+- `{current}` - Current version.
+- `{latest}` - Latest version.
+- `{command}` - Install command.
+
+```js
+notifier.notify({message: 'Run `npm install {name}@{latest}` to update.'});
```

Comment:
This is not a good example as people should use the `{updateCommand}` placeholder so it works with Yarn too.